server/src/main/scala/passengersplits/core/PassengerInfoRouterActor.scala

```diff
@@ -70,6 +70,7 @@ class PassengerSplitsInfoByPortRouter extends
   Actor with PassengerQueueCalculator with ActorLogging
   with SimpleRouterActor[PassengerInfoRouterActor] {
 
+  var latestFileName: Option[String] = None
```

Comment:
I can't see if/where this is used apart from being set a few lines down. Is it used?